### PR TITLE
cleaner code_for_op order [pr]

### PR DESCRIPTION
**Pull request by ignaciosica**
Just a reorder of the items for cleaner look

### CODE REVIEW

File: tinygrad/renderer/cstyle.py [hunk 1]
```diff
@@ -81,17 +81,13 @@ class CStyleLanguage(Renderer):
...
-    UnaryOps.RECIP: lambda x,dtype: f"(1/{x})",
-    UnaryOps.NEG: lambda x,dtype: f"-{x}",
-    UnaryOps.EXP2: lambda x,dtype: f"exp2({x})", UnaryOps.LOG2: lambda x,dtype: f"log2({x})", UnaryOps.SIN: lambda x,dtype: f"sin({x})",
-    BinaryOps.SHL: lambda a,b,dtype: f"({a}<<{b})", BinaryOps.SHR: lambda a,b,dtype: f"({a}>>{b})",
-    BinaryOps.ADD: lambda a,b,dtype: f"({a}+{b})", BinaryOps.SUB: lambda a,b,dtype: f"({a}-{b})",
-    BinaryOps.IDIV: lambda a,b,dtype: f"({a}/{b})", BinaryOps.MUL: lambda a,b,dtype: f"({a}*{b})", BinaryOps.MOD: lambda a,b,dtype: f"({a}%{b})",
-    BinaryOps.CMPLT: lambda a,b,dtype: f"({a}<{b})", BinaryOps.CMPNE: lambda a,b,dtype: f"({a}!={b})", BinaryOps.XOR: lambda a,b,dtype: f"({a}^{b})",
-    BinaryOps.AND: lambda a,b,dtype: f"({a}&{b})", BinaryOps.OR: lambda a,b,dtype: f"({a}|{b})",
-    TernaryOps.WHERE: lambda a,b,c,dtype: f"({a}?{b}:{c})"}
+  code_for_op: Dict = { TernaryOps.WHERE: lambda a,b,c,dtype: f"({a}?{b}:{c})",
```
Comment (chenyuxyz) [inline tinygrad/renderer/cstyle.py:84]: i like the original order that goes unary -> binary -> ternary. and put similar things together like (SHL, SHR), (ADD, SUB, MUL, IDIV, MOD) (CMPLT, CMPNE) etc

File: tinygrad/renderer/cstyle.py [hunk 2]
```diff
@@ -311,12 +308,12 @@ def __reduce__(self): return self.__class__, (self.arch,)
   float4 = "make_float4"
   code_for_workitem = {"g": lambda x: f"blockIdx.{chr(120+int(x))}", "l": lambda x: f"threadIdx.{chr(120+int(x))}",
                        "i": lambda x: f"(blockIdx.{chr(120+int(x))}*blockDim.{chr(120+int(x))}+threadIdx.{chr(120+int(x))})"}
-  code_for_op = {**CStyleLanguage.code_for_op,
-    UnaryOps.RECIP: lambda x,dtype: f"hrcp({x})" if dtype in (dtypes.half, dtypes.bfloat16) else f"(1/{x})",
-    UnaryOps.SQRT: lambda x,dtype: f"hsqrt({x})" if dtype in (dtypes.half, dtypes.bfloat16) else f"sqrt({x})",
+  code_for_op = { **CStyleLanguage.code_for_op,
```
Comment (chenyuxyz) [inline tinygrad/renderer/cstyle.py:311]: any reason to reorder these (and the one below)?

Comment (ignaciosica) [reply tinygrad/renderer/cstyle.py:311]: just for readability, sin, they are visually aligned and stay coherently ordered

Comment (ignaciosica) [reply tinygrad/renderer/cstyle.py:311]: > just for readability, they are visually aligned and stay coherently ordered

same as for cstyle, no functional reason either, just thought it ended up cleaner

Comment (chenyuxyz) [reply tinygrad/renderer/cstyle.py:311]: oh it's sorted by name length... fine i guess

**Pull request merged**